src/variables/components/VariablesTab.test.tsx

```diff
@@ -0,0 +1,427 @@
+      message: 'Variable deleted successfully',
+      data: {
+        name: 'test_variable_name',
+        id: 'test_variable_id',
+        orgID: 'test_org_id',
+        arguments: {
+          type: 'map',
+          values: {
+            sample: 'value',
+          },
+        },
+      },
+    }
+  }),
+}))
+
+jest.mock('src/client/index.ts')
+jest.mock('src/shared/actions/notifications')
+jest.mock('src/resources/components/GetResources')
+jest.mock(
+  'src/annotations/components/overlay/CreateAnnotationStreamOverlay',
+  () => () => null
+)
+jest.mock('src/checks/components/NewThresholdCheckEO.tsx', () => () => null)
+jest.mock('src/checks/components/NewDeadmanCheckEO.tsx', () => () => null)
+jest.mock('src/resources/selectors/index.ts', () => {
+  return {
+    getAll: jest.fn(() => {
+      return []
+    }),
+    getLabels: jest.fn(() => {
+      return []
+    }),
+  }
+})
+
+// Mock State
+import {renderWithReduxAndRouter} from 'src/mockState'
+import {withRouterProps} from 'mocks/dummyData'
+import {mockAppState} from 'src/mockAppState'
+
+// Redux
+import VariablesIndex from '../containers/VariablesIndex'
+import {normalize} from 'normalizr'
+import {Organization} from '../../client'
+import {OrgEntities, RemoteDataState} from '../../types'
+import {arrayOfOrgs} from '../../schemas'
+import {fireEvent, cleanup, screen, waitFor} from '@testing-library/react'
+import {
+  OverlayController,
+  OverlayProviderComp,
+} from '../../overlays/components/OverlayController'
+
+import {mocked} from 'ts-jest/utils'
+import {notify} from 'src/shared/actions/notifications'
+import {
+  createVariableSuccess,
+  deleteVariableSuccess,
+  updateVariableSuccess,
+} from '../../shared/copy/notifications'
+
+const defaultProps: any = {
+  ...withRouterProps,
+  org: '',
+}
+
+const setup = (props = defaultProps) => {
+  return renderWithReduxAndRouter(
+    <>
+      <OverlayProviderComp>
+        <OverlayController />
+        <VariablesIndex {...props} />
+      </OverlayProviderComp>
+    </>,
+    _fakeLocalStorage => {
+      const appState = {...mockAppState}
+      return appState
+    }
+  )
+}
+
+let getByTestId
+let store
+let getByTitle
+
+describe('the variable integration tests', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+
+    const renderResult = setup()
+
+    store = renderResult.store
+    getByTestId = renderResult.getByTestId
+    getByTitle = renderResult.getByTitle
+
+    const org = {name: 'test_org_name', id: 'test_org_id'}
+
+    store.dispatch({
+      type: 'SET_ORG',
+      org: org,
+    })
+    const organizations = normalize<Organization, OrgEntities, string[]>(
+      [org],
+      arrayOfOrgs
+    )
+    store.dispatch({
+      type: 'SET_ORGS',
+      schema: organizations,
+      status: RemoteDataState.Done,
+    })
+  })
+
+  afterEach(() => {
+    cleanup()
+  })
+
+  describe('handling the variable creation process', () => {
+    it('Create a Variable of Map type', async () => {
```

Comment:
```suggestion
    it('can create a Variable of Map type', async () => {
```

try to read the `it` like part of a sentence:

>it create a variable of Map type

vs 

> it **can** create a variable of Map type

the text should also flow smoothly with the `describe text`, eg:

>the variable creation process ... can create a variable of Map type